charts/galley/templates/configmap.yaml

```diff
@@ -43,6 +43,7 @@ data:
       intraListing: false
       maxTeamSize: {{ .settings.maxTeamSize }}
       maxConvSize: {{ .settings.maxConvSize }}
+      truncationLimit: {{ .settings.truncationLimit }}
```

Comment:
It will make upgrades easier if we default this to 500 like the `maxTeamSize`.

Reply:
Actually this is optional, and in general nobody should set it. Maybe we can adjust it to:
```
{{- if .settings. truncationLimit }}
    truncationLimit: {{ .settings.truncationLimit }}
{{- end }}
```

Reply:
Yeah, that is even better.

Reply:
https://github.com/wireapp/wire-server-deploy/pull/231/commits/2c4a00d6854a48074f812fa521ffe81351522f86